news/2025/2025-12-05-introducing-quick-play.md

```diff
@@ -0,0 +1,61 @@
+
+We hope you'll join us this coming November 29th in a stress-test with the addition of over 250 new maps (including mods) to the pool, global rating leaderboards, randomised voting, and more. Of course, the team will also be participating!
+
+## But what exactly is quick play?
+
+For those who still haven't checked the latest update video, here's a quick rundown of what quick play is by the hands of [peppy](https://osu.ppy.sh/users/2) and [smoogipoo](https://osu.ppy.sh/users/1040328) — the minds behind quick play as a whole!
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/YKUQ0Y_pHaI?start=0&end=307" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Quick play is an attempt at branching out into more ways for players to play and compete in the game. As opposed to leaderboards and [performance points](/wiki/Performance_points) which allow players to compete in a global environment, quick play offers a much more direct approach to competition by facing players against each other to test their mettle! In turn, quick play allows players to measure their skill relatively to their peers, while providing a much more direct way to interact with the community — not dissimilar to how multiplayer lobbies work.
+
+The name of the game lies in its name: *quick* play! The system aims to provide a very casual experience for players who just want to hop into a lobby and get into the nitty-gritty of things. No need to create a lobby, wait for players to join, wait for everyone to ready up, and manually rotate the host between players. You get the same experience at a much quicker and steady pace from a selection of pre-curated mappools across all game modes!
+
+More about the project's inner workings can be found in the [wiki article](/wiki/Gameplay/Quick_play) for the curious.
+
+## Quick play launch event
+
+In an attempt to get more players to try out quick play, the devs have planned four weeks of test runs. These runs will test various versions of the system, and allow the team to gather feedback for its general direction and any future adjustments.
+
+Each week, the matchmaking system will switch between **1v1** and **free-for-all (FFA)** battles, and mappools that are either **manually curated** or **randomised** between all Ranked maps:
+
+| Dates | Pool type | Matchmaking type |
+| :-: | :-: | :-: |
+| December 6th – December 13th | Manually curated | 1v1 |
+| December 13th – December 20th | Manually curated | FFA |
+| December 20th – December 27th | Randomised | 1v1 |
+| December 27th – January 3rd | Randomised | FFA |
+
+While the manually curated pools are an expansion of the existing pools players have been able to play so far, we want to experiment with pseudo-randomised pools that would be created on the fly based on a lobby's average skill level (i.e. matchmaking rating or MMR). We'll also be checking how people feel about different lobby sizes.
```

Comment:
Here too, specifically:

> based on a lobby's average skill level

Because both sets of pools are randomised and filtered down to the average skill level, the only difference is one will be randomising across all FA maps.